app/views/portal/show.rb

```diff
@@ -783,154 +753,173 @@ def iiif_manifesto(identifier, collection)
 
-        if media_type == 'text' && mime_type == 'text/plain; charset=utf-8'
-          item[:playable] = false
-          item[:downloadable] = false
-        end
+    def item_technical_metadata(web_resource, mime_type)
+      file_size = number_to_human_size(render_document_show_field_value(web_resource, 'ebucoreFileByteSize')) || ''
+      {
+        mime_type: mime_type,
+        file_size: file_size.split(' ').first,
+        file_unit: file_size.split(' ').last,
+        codec: render_document_show_field_value(web_resource, 'edmCodecName'),
+        width: render_document_show_field_value(web_resource, 'ebucoreWidth'),
+        height: render_document_show_field_value(web_resource, 'ebucoreHeight'),
+        size_unit: 'pixels',
+        runtime: render_document_show_field_value(web_resource, 'ebucoreDuration'),
+        runtime_unit: 'seconds'
+      }
+    end
 
-        if media_type == 'video' && mime_type == 'text/plain; charset=utf-8'
-          item[:playable] = false
-          item[:downloadable] = false
-        end
+    def item_playable?(web_resource_url, mime_type, media_type)
+      if web_resource_url.blank? ||
+        mime_type.blank? ||
+        (mime_type == 'video/mpeg') ||
+        (media_type == 'text' && mime_type == 'text/plain; charset=utf-8') ||
+        (media_type == 'video' && mime_type == 'text/plain; charset=utf-8')
+        false
+      else
+        true
+      end
+    end
 
-        if media_type == 'image'
-          item['is_image'] = true
-          players << { image: true }
-
-          # we only have a thumbnail for the first
-          # - full image needed for the others
-          # - metadata service needed
-          # if web_resource_url != edm_resource_url
-          #   item[:thumbnail] = web_resource_url
-          # end
-        elsif media_type == 'audio' || media_type == 'sound'
-          item['is_audio'] = true
-          players << { audio: true }
-          item[:thumbnail] = item[:thumbnail] || 'http://europeanastatic.eu/api/image?size=BRIEF_DOC&type=SOUND'
-        elsif media_type == 'iiif'
-          item['is_iiif'] = true
-          players << { iiif: true }
-        elsif media_type == 'pdf'
-          item['is_pdf'] = true
-          players << { pdf: true }
-        elsif media_type == 'text'
-          if mime_type == 'application/pdf'
-            item['is_pdf'] = true
-            players << { pdf: true }
-          else
-            item['is_text'] = true
-          end
-        elsif media_type == 'video'
-          item['is_video'] = true
-          players << { video: true }
-        else
-          item['is_unknown_type'] = render_document_show_field_value(web_resource, 'about')
-        end
+    def item_downloadable?(web_resource_url, mime_type, media_type, media_rights)
+      if web_resource_url.blank? ||
+        mime_type.blank? ||
+        download_disabled?(media_rights) ||
+        (media_type == 'text' && mime_type == 'text/plain; charset=utf-8') ||
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.